packages/dds/tree/src/test/simple-tree/object.spec.ts

```diff
@@ -323,8 +311,8 @@ const factory = new SchemaFactory("test");
 
 describe("Object-like", () => {
 	describe("setting an invalid field", () => {
-		// TODO: Restore original behavior for bare '_.object()'?
-		// https://dev.azure.com/fluidframework/internal/_workitems/edit/6549
+		// TODO: Disallow setting out-of-schema properties?
+		// https://dev.azure.com/fluidframework/internal/_workitems/edit/7177
```

Comment:
I haven't pulled this thread yet either, but suspect that the Proxy was intercepting and preventing calls to subclass members, prevent calls to things that users declare on their classes.